tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -38,296 +38,255 @@
 from gt4py.next.iterator.embedded import index_field, np_as_located_field
 from gt4py.next.program_processors.runners import gtfn_cpu
 
+from next_tests.integration_tests.feature_tests import cases
+from next_tests.integration_tests.feature_tests.cases import (
```

Comment:
I would violate our style guides even more, but I am probably alone
`from next_tests.integration_tests.feature_tests.cases import *`

Reply:
I am willing to do this only if the style guide is changed (could allow for this in tests or other well defined circumstances).

Reply:
I disagree. I don't think star imports are a good idea except in quick interactive prototyping sessions.